src/types.rs

```diff
@@ -54,11 +54,51 @@ impl From<&'static str> for Error {
 #[derive(Deserialize, Serialize)]
 pub struct AccountCredentials {
     pub(crate) id: String,
-    pub(crate) key_pkcs8: String,
+    /// Stored in DER, serialized as base64
+    #[serde(with = "pkcs8_serde")]
+    pub(crate) key_pkcs8: Vec<u8>,
```

Comment:
I agree that `key_pkcs8` is technically a pleonasm. However, in code, I think the context of how the key is encoded is useful, for example in code like `Key::from_pkcs8_der(credentials.key_pkcs8.as_ref())?` I find it quite helpful to see that the function name expectation matches the data I'm passing in, since expressing these kinds of contraints in the type system (like `Pkcs8(Der(private_key))`) gets a bit cumbersome and is probably overkill in this case.